src/main/features/paid-in-full/form/models/datePaid.ts

```diff
@@ -0,0 +1,37 @@
+import { IsValidLocalDate } from '@hmcts/cmc-validators'
+import { ValidateNested } from 'class-validator'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotInFuture } from 'forms/validation/validators/notInFuture'
+export class ValidationErrors {
```

Comment:
Please consider if we can reuse the error messages from common place (`src/main/app/forms/validation/validationErrors.ts`)

Reply:
Changed to use common validation errors class.